src/app/globals.css

```
@@ -3,6 +3,12 @@
 @layer base {
   html {
     @apply text-black;
+    background: #1e2129;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)